packages/web/src/pages/rewards-page/components/ChallengeRewards/RewardPanel.tsx

```diff
@@ -33,7 +33,7 @@ import { StatusPill } from './StatusPill'
 const { getOptimisticUserChallenges } = challengesSelectors
 
 const PANEL_HEIGHT = 200
-const PANEL_WIDTH = 336
+const PANEL_WIDTH = 320
```

Comment:
thought i fixed something similar in this PR but maybe this is different?
https://github.com/AudiusProject/audius-protocol/commit/ea9eaa947092c6a206b7da8e9a702095ca37c333

Reply:
yeah a bit different